lte/gateway/python/magma/pipelined/qos/qos_tc_impl.py

```diff
@@ -36,9 +37,8 @@ def argSplit(cmd: str) -> List[str]:
 def run_cmd(cmd_list, show_error=True) -> int:
     err = 0
     for cmd in cmd_list:
-        LOG.debug("running %s", cmd)
+        LOG.info("running %s", cmd)
```

Comment:
is this going to spam like crazy?

Reply:
it shld not spam much, just to be safe, I changed it to debug.